yelp_package/Makefile

```diff
@@ -21,16 +21,26 @@ UID:=`id -u`
 GID:=`id -g`
 DOCKER_RUN_LUCID:=docker run -t -v  $(CURDIR)/../:/work:rw paastatools_lucid_container
 DOCKER_RUN_TRUSTY:=docker run -t -v  $(CURDIR)/../:/work:rw paastatools_trusty_container
-ADD_MISSING_DEPS:=-diff --unchanged-line-format= --old-line-format= --new-line-format='%L' ../requirements.txt ./extra_requirements.txt >> ../requirements.txt
+ifeq ($(PAASTA_ENV),YELP)
+	ADD_MISSING_DEPS_MAYBE:=-diff --unchanged-line-format= --old-line-format= --new-line-format='%L' ../requirements.txt ./extra_requirements_yelp.txt >> ../requirements.txt
+	ADD_YELP_PREFIX_MAYBE:=dch -v $(RELEASE)-yelp1 --changelog ../debian/changelog $$'Build for yelp: add scribereader to virtualenv'
+else
+	ADD_MISSING_DEPS_MAYBE:=true
+	ADD_YELP_PREFIX_MAYBE:=true
```

Comment:
I realize you are trying to treat these as noops, but to make it more obvious, how about defining a `NOOP` variable with a value of `true` and then set `ADD_MISSING_DEPS_MAYBE` and `ADD_YELP_PREFIX_MAYBE` equal to that?